regions/plotting/plotting.py

```diff
@@ -0,0 +1,58 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.coordinates import SkyCoord
+from regions.io.read_ds9 import read_ds9
+from regions.shapes.circle import CircleSkyRegion
+import astropy.units as u
+
+
+def region_object_to_mpl_artist(region_list, ax=None):
+    """Convert a astropy region to a matplotlib artist"""
+    ids = {"<class 'regions.shapes.circle.CircleSkyRegion'>": 'skycircle',
+           "<class 'regions.shapes.circle.CirclePixelRegion'>": 'pixcircle',
+           "<class 'regions.shapes.ellipse.EllipseSkyRegion'>": 'skyellipse',
+           "<class 'regions.shapes.ellipse.EllipsePixelRegion'>": 'pixellipse',
+           "<class 'regions.shapes.polygon.PolygonSkyRegion'>": 'skypolygon',
+           "<class 'regions.shapes.polygon.PolygonPixelRegion'>": 'pixpolygon'}
+
+    output = list()
+
+    for reg in region_list:
+        temp = str(reg.__class__)
```

Comment:
you could just use the region class directly, rather than the string name of the class